src/Uno.Extensions.Reactive/Core/ListState.T.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using System.Collections.Immutable;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Uno.Extensions.Reactive;
+
+internal record ListState<T>(IState<IImmutableList<T>> implementation) : ListFeedImpl<T>(implementation), IListState<T>
+{
+	private readonly IState<IImmutableList<T>> implementation = implementation;
```

Comment:
That's a weird pattern... Why can't `Implementation` be made public?

Reply:
To avoid hacky usages ... but this class will be removed / refactored in my next PR (we need to integrate more deeply in the implementation `State` to track the changes), so I'll keep it like that for now.